packages/nouns-contracts/test/foundry/NounsDAOLogicV3/ProposeBySigs.t.sol

```diff
@@ -440,6 +440,34 @@ contract ProposeBySigsTest is NounsDAOLogicV3BaseTest {
         assertEq(proposal.signers, expectedSigners);
     }
 
+    function test_givenProposerWithNoVotesAndTwoSignaturesBySameSigner_reverts() public {
+        // Minting to push a single signer below threshold
+        vm.startPrank(minter);
+        for (uint256 i = 0; i < 6; ++i) {
+            nounsToken.mint();
+        }
+        vm.roll(block.number + 1);
+        vm.stopPrank();
+
+        NounsDAOV3Proposals.ProposalTxs memory txs = makeTxs(makeAddr('target'), 0, '', '');
+        uint256 expirationTimestamp = block.timestamp + 1234;
+        NounsDAOStorageV3.ProposerSignature[] memory proposerSignatures = new NounsDAOStorageV3.ProposerSignature[](2);
+        proposerSignatures[0] = NounsDAOStorageV3.ProposerSignature(
+            signProposal(proposerWithNoVotes, signerWithVote1PK, txs, 'description', expirationTimestamp, address(dao)),
+            signerWithVote1,
+            expirationTimestamp
+        );
+        proposerSignatures[1] = NounsDAOStorageV3.ProposerSignature(
+            signProposal(proposerWithNoVotes, signerWithVote1PK, txs, 'description', expirationTimestamp, address(dao)),
+            signerWithVote1,
+            expirationTimestamp
```

Comment:
seems perhaps important to use a different timestamp, maybe as a separate test?
in this test I believe both sigs will have the same bytes, but the actual test is that it fails given the same signer.

wdyt?

Reply:
sure, I changed the timestamp